commons/libs/constants/ApiResponseMessages.ts

```diff
@@ -0,0 +1,4 @@
+export const CREATE_DONATION_REQUEST_SUCCESS = 'We have accepted your request, and we will let you know when we find a donors.'
```

Comment:
find donors!

Reply:
Updated in f98450b71743d5473aa397d4854e7347002cc21c